code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -53,12 +53,29 @@
 	ammo_type = "/obj/item/ammo_casing/a12mm/assault"
 
 /obj/item/ammo_storage/box/c45
-	name = "ammo box (.45)"
+	name = "Pistol ammo box (.45)"
```

Comment:
Capitilization is used for \the stuff. If it is capitalized, it skips adding a the to it

>Anon is hit by the ammo box (.45)
>Anon is hit by Pistol ammo box (.45)